dtmi/vecow/AIC-110

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
File name should be `aic_110-1.json`.

Reply:
I have modified and re-uploaded the file.

Reply:
It appears the file name change did not make it in this PR.